bench/workgen/validate_mirror_tables.py

```diff
@@ -56,14 +56,15 @@ def get_wiredtiger_db_files(dir):
 # Given a list of database tables, return a list of mirrored uri pairs.
 def get_mirrors(db_dir, db_files):
 
-    mirrors = []
+    connection = wiredtiger_open(db_dir, 'readonly')
     db_files_remaining = set(db_files)
+    mirrors = []
 
     for filename in db_files:
         if filename not in db_files_remaining:
             continue
         db_files_remaining.remove(filename)
-        mirror_filename = get_mirror_file(db_dir, f'table:{filename}')
```

Comment:
Should the `get_wiredtiger_db_files` be fetching all the files from the metadata instead? Since it is possible that disk files are not in sync with the metadata, maybe we use the metadata as our source of truth.

Reply:
I agree with you, I have implemented this approach in cb9806fbe8a4dd27435d750e47391961865fc7bb.